packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/__tests__/recent.spec.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { mockStore, StoreProvider } from '@deriv/stores';
+import { render, screen } from '@testing-library/react';
+import { isMobile } from '@deriv/shared';
+import { mock_ws } from 'Utils/mock';
+import RootStore from 'Stores/root-store';
+import { DBotStoreProvider, mockDBotStore, useDBotStore } from 'Stores/useDBotStore';
+import RecentComponent from '../recent';
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    isMobile: jest.fn(() => false),
+}));
+
+jest.mock('@deriv/bot-skeleton/src/scratch/blockly', () => jest.fn());
+
+jest.mock('@deriv/bot-skeleton/src/scratch/dbot', () => ({}));
+
+jest.mock('@deriv/bot-skeleton/src/scratch/hooks/block_svg', () => jest.fn());
+
+jest.mock('Stores/useDBotStore');
```

Comment:
@maryia-matskevich-deriv do we need to mock this `useDBotStore`?

Reply:
@shafin-deriv if you comment on this line you can see that all tests will fall, if you know how to improve it please suggest. I'm always open to improvements😌

Reply:
@shafin-deriv improved✅